spec/ParseUser.spec.js

```diff
@@ -4430,3 +4430,72 @@ describe('allowClientClassCreation option', () => {
     defaultConfiguration.allowClientClassCreation = true;
   });
 });
+
+describe('custom log levels for username already exists error', () => {
```

Comment:
This test setup contains a lot of duplicate code. Simply this to a single test with a loop